### PR TITLE
Add readmes for UNet/ICNet trained on CamVid

### PR DESCRIPTION
**Pull request by vshampor**

### CODE REVIEW

**Comment by vshampor** [comment]
@AlexKoff88

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
please retarget your PR to develop branch

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
well, you also need rebase your branch to develop because path to models has changed between master and develop branches and you need to add links to your models in index.md file under models/intel.
Also model.yml file is required, please look at other intel models in develop branch for details

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
check failed: icnet-camvid-int8-sparse-v2-0001 not in index.md file

**Comment by eaidova** [comment]
@vshampor, please move added readme files from` intel_models/<model_name>` to `models/intel/<model_name>`

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Input
+
+Image, shape - `1,3,768,960`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`
```
Comment (eaidova) [inline intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: model converted with `--reverse_input_channels` flag so channels order is `BGR`

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The model input is a blob that consists of a single image of "1x3x768x960" in RGB order. The pixel values are floating-point values that were obtained by taking each channel value as an integer in the [0, 255] range, dividing it by 255, and the normalizing the results given a mean of [0.39068785, 0.40521392, 0.41434407] and standard deviation of [0.29652068, 0.30514979, 0.30080369] (RGB order).
+
+The model output for `icnet-camvid-fp32-0001` is the per-pixel probabilities of each input pixel belonging to one of the 12 classes of the CamVid dataset.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| Type              | Semantic segmentation |
```
Comment (eizamaliev) [inline intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: Please, provide MParams and GFlops values, if avaible.

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+| Source framework  | PyTorch               |
+
+## Accuracy
+
+The quality metrics calculated on CamVid validation dataset is 67.89% mIoU.
```
Comment (eaidova) [inline intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: Why do you specify metric value twice? The table with the result and specification on which dataset it was calculated, will be enough.

Also, on my view, specification how mean IOU was calculated also will not be superfluous (e.g.like [here](https://github.com/opencv/open_model_zoo/blob/develop/models/intel/semantic-segmentation-adas-0001/description/semantic-segmentation-adas-0001.md#accuracy))

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md (hunk 1 above)
Comment (eizamaliev) [inline intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: By default, channel order must be `BGR`.

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+The model input is a blob that consists of a single image of "1x3x768x960" in RGB order. The pixel values are floating-point values that were obtained by taking each channel value as an integer in the [0, 255] range, dividing it by 255, and the normalizing the results given a mean of [0.39068785, 0.40521392, 0.41434407] and standard deviation of [0.29652068, 0.30514979, 0.30080369] (RGB order).
+
+The model output for `icnet-camvid-fp32-0001` is the per-pixel probabilities of each input pixel belonging to one of the 12 classes of the CamVid dataset.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| Type              | Semantic segmentation |
+| Source framework  | PyTorch               |
```
Comment (eizamaliev) [inline intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]:
```suggestion
| Source framework  | PyTorch\*               |
```

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md (hunk 3 above)
Comment (eizamaliev) [inline intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: Please, provide link to dataset or legal information about it, if it external

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# icnet-camvid-fp32-0001
```
Comment (eizamaliev) [inline intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: Add `Legal information` part (see other model's description for example)

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md (hunk 2 above)
Comment (vshampor) [reply intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: I suppose these should be gathered from the benchmarks, but I could not find the results for the models in this MR in the latest benchmarks.

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md (hunk 1 above)
Comment (vshampor) [reply intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: Done

Comment (vshampor) [reply intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: Done

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md (hunk 4 above)
Comment (vshampor) [reply intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: Done

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md (hunk 3 above)
Comment (vshampor) [reply intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: Done

File: intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md (hunk 5 above)
Comment (vshampor) [reply intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md]: Done

**Comment by vshampor** [comment]
To be continued in #360 - cannot retarget the PR otherwise to pick the develop branch of my fork as source.

**Pull request closed without merging**